design/features/early-access/README.md

```diff
@@ -0,0 +1,70 @@
+# Early access journey 
+
+## Status
+
+Date created: *2023-11-06*  
+
+- [Trello card](https://trello.com/c/I8HETm57/1041-design-the-early-access-upgrade-mou-journey-ready-for-user-research "Trello card")
+- [Mural working board](https://app.mural.co/t/gaap0347/m/gaap0347/1691659258399/af093136c3f86be05860120cb92d6810e98f3643?sender=u61ceb93b0f861033ca7c7448 "Mural working board")
+
+___
+
+## Contents
+
+- [Early access journey](#early-access-journey)
+  - [Status](#status)
+  - [What](#what)
+  - [Problems we tried to solve](#Problems-we-tried-to-solve)
+  - [User needs](#user-needs)
+  - [Changes](#changes)
+  - [What we did](#what-we-did)
+
+___
+<br>
+
+## What
+This journey is for trial users requesting to become editor so that they can start publishing their forms on GOV.UK. 
+
+<br>
+
+## Problems we tried to solve
+* There is currently no process in place where users can request to become an editor, sign mou and go through requirements check to become an editor.
+* There is also no process in place for adoption to receive requests for editor acccess
+
+<br>
+
+## User needs: 
+ * As a trial user I need to know what are the criteria to become an editor
```

Comment:
Made an edit to this user need 

"As a trial user I need to know what are the criteria and steps to become an editor"